go.mod

```diff
@@ -47,6 +47,7 @@ require (
 	github.com/santhosh-tekuri/jsonschema/v5 v5.3.1
 	github.com/sqlc-dev/pqtype v0.3.0
 	github.com/swaggest/jsonschema-go v0.3.72
+	github.com/tbd54566975/web5-go v0.24.0
```

Comment:
It looks like this was added just to get a type to use for testing, but this will slow down the builds, so let's remove it and use some type in the stdlib or something.

Reply:
Is it ok to keep using it in the testdata project (so it is not in the main go.mod, just the test project used in integration tests). 

Using this class actually showed up some real world issues as it has a custom serialization format, and was supported in both languages via third party libraries.

Reply:
Yeah that's fine!